### PR TITLE
1959 Badge tests added

### PR DESCRIPTION
**Pull request by MCdanja**

### CODE REVIEW

**Comment by mf-code** [comment]
[5763] - f0uFUfu9MM4G

**Comment by mf-code** [comment]
[5763] - VHY7qgSdwVcf

**Comment by mf-code** [comment]
[5763] - emIfygyFehjK

**Comment by mf-code** [comment]
[5763] - ixN5mcmBiTEt

**Comment by mf-code** [comment]
[5763] - N7u8QClkYSpu

**Comment by mf-code** [comment]
[5763] - H4VnTzTVjUzj

**Comment by mf-code** [comment]
[5763] - HgpIyxSK3QO8

**Comment by mf-code** [comment]
[5763] - uFmrkjCJZp8q

**Comment by mf-code** [comment]
[5763] - unTHtNWTGLHE

**Comment by mf-code** [comment]
[5763] -

**Comment by mf-code** [comment]
[5763] - r0fYMGc7X6KS

**Comment by mf-code** [comment]
[5763] - Rc8PFLgkbJcP

**Comment by mf-code** [comment]
[5763] - 40F2vWoZEg3E

**Comment by mf-code** [comment]
[5763] - h6BFW6q5SiXu

**Comment by mf-code** [comment]
[5763] - openjdk8 - Allure report on Netlify: https://5f083a9a8d291b4836a5e3f7--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5763] - openjdk10 - Allure report on Netlify: https://5f083b595edcd5a1e7a72fa4--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @Test
+    public void basicBadgeTest() {
+        badgeSection.badgeWithText.is().displayed();
+        badgeSection.badgeWithButton.is().displayed();
+        badgeSection.badgeWithIcon.is().displayed();
+    }
+
+    @Test
+    public void badgeTextTest() {
+        badgeSection.badgeWithText.isDisplayed();
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java:28]: What is the reason for calling the method isDisplayed() here and in others tests?

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        badgeSection.badgeWithButton.isDisplayed();
+        badgeSection.badgeWithButton.has().text("8");
+
+        badgeSection.badgeWithIcon.isDisplayed();
+        badgeSection.badgeWithIcon.has().text("15");
+
+    }
+
+    @Test
+    public void hiddenMessageTest() {
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java:40]: There are too many assets in one method.

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java (hunk 1 above)
Comment (MCdanja) [reply jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java]: Fixed

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java (hunk 2 above)
Comment (MCdanja) [reply jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BadgeTests.java:34]: Fixed

**Comment by mf-code** [comment]
[5775] - nhHiuyWt96Oi

**Comment by mf-code** [comment]
[5775] - 1sXv5OlQXEg3

**Comment by mf-code** [comment]
[5775] - MGzTn7kggi5g

**Comment by mf-code** [comment]
[5775] - VO1eIYWFZ5vX

**Comment by mf-code** [comment]
[5775] - Fmf7pCkhcO2j

**Comment by mf-code** [comment]
[5775] - Baw4IYypdyN7

**Comment by mf-code** [comment]
[5775] - 9fYb2uQE4xpN

**Comment by mf-code** [comment]
[5775] - vWdeGWpjUPoN

**Comment by mf-code** [comment]
[5775] - UIFjFsPzMesO

**Comment by mf-code** [comment]
[5775] - 2VF2X49WN2Gj

**Comment by mf-code** [comment]
[5775] -

**Comment by mf-code** [comment]
[5775] -

**Comment by mf-code** [comment]
[5775] - I13C2OlXuAa8

**Comment by mf-code** [comment]
[5775] - vPaM2OxjfaZW

**Comment by mf-code** [comment]
[5775] - openjdk8 - Allure report on Netlify: https://5f0c2ee0dcb30f66f9dbbecc--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5775] - openjdk10 - Allure report on Netlify: https://5f0c2f87682c2f5f736f3c7b--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5781] - Hh9GJtNn8mt0

**Comment by mf-code** [comment]
[5781] - tLsNmXgC4Qri

**Comment by mf-code** [comment]
[5781] - AXvVFwddpAie

**Comment by mf-code** [comment]
[5781] - 2heC25hNqUR1

**Comment by mf-code** [comment]
[5781] - 9jqiODtguXUc

**Comment by mf-code** [comment]
[5781] - ItBzXTd5F3Hv

**Comment by mf-code** [comment]
[5781] -

**Comment by mf-code** [comment]
[5781] -

**Comment by mf-code** [comment]
[5781] - Xu1xWnEfpz6U

**Comment by mf-code** [comment]
[5781] - W0LZl3uji61P

**Comment by mf-code** [comment]
[5781] - ii1D8WK64oxp

**Comment by mf-code** [comment]
[5781] - kAsi2iIevWNB

**Comment by mf-code** [comment]
[5781] - oMn5FLmSofDX

**Comment by mf-code** [comment]
[5781] - dz73WPqw2N4u

**Comment by mf-code** [comment]
[5781] - openjdk8 - Allure report on Netlify: https://5f0c5688fa9b88d503b74a2b--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5781] - openjdk10 - Allure report on Netlify: https://5f0c572d82c1436fb383b3d2--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5782] - X50BRssWaUM7

**Comment by mf-code** [comment]
[5782] - N62FVXGVuVKj

**Comment by mf-code** [comment]
[5782] - VWrgT9zyzTi0

**Comment by mf-code** [comment]
[5782] - ewMUqtPNvtQR

**Comment by mf-code** [comment]
[5782] -

**Comment by mf-code** [comment]
[5782] - gE7Fx6qsmwR3

**Comment by mf-code** [comment]
[5782] - 9RX9KOx8IpWg

**Comment by mf-code** [comment]
[5782] - 9V3VAMA57hsk

**Comment by mf-code** [comment]
[5782] - 0jZGGeCZwuj8

**Comment by mf-code** [comment]
[5782] - 1bsEhApenxQL

**Comment by mf-code** [comment]
[5782] - o5CRUx0ycqF6

**Comment by mf-code** [comment]
[5782] -

**Comment by mf-code** [comment]
[5782] - vWoa7AXrRrNU

**Comment by mf-code** [comment]
[5782] - rQKuGy3aeGX2

**Comment by mf-code** [comment]
[5782] - openjdk8 - Allure report on Netlify: https://5f0c677d45f6a6a2fe4eee50--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5782] - openjdk10 - Allure report on Netlify: https://5f0c685d84abef74e956f669--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5782] - PgH432YJkPvU

**Comment by mf-code** [comment]
[5782] - 0Cto5djp4nJd

**Comment by mf-code** [comment]
[5782] - 9H2O6rPaJNcK

**Comment by mf-code** [comment]
[5782] - FXTVRr61Y6lU

**Comment by mf-code** [comment]
[5782] - 8gezsqqJwgN7

**Comment by mf-code** [comment]
[5782] - B5Z33DIqLnc3

**Comment by mf-code** [comment]
[5782] - CxG1y0k4Kagr

**Comment by mf-code** [comment]
[5782] - khnxRXaXHDag

**Comment by mf-code** [comment]
[5782] - kwgpP94x06Vr

**Comment by mf-code** [comment]
[5782] - DRbGm8hzY6oM

**Comment by mf-code** [comment]
[5782] - RMB4TPg0EpT7

**Comment by mf-code** [comment]
[5782] - iZ0hKxOtIxI7

**Comment by mf-code** [comment]
[5782] - DkeoeuneGMjW

**Comment by mf-code** [comment]
[5782] - 7fYOcM4z3lrC

**Comment by mf-code** [comment]
[5782] - openjdk8 - Allure report on Netlify: https://5f0ec962d860c1f0269d7cc7--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5782] - openjdk10 - Allure report on Netlify: https://5f0eca4e5550e98afcf5d382--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5797] - 8maoDlYMGIKc

**Comment by mf-code** [comment]
[5797] - ZfYxBHGbXGaH

**Comment by mf-code** [comment]
[5797] - d0C49OM1SBZ4

**Comment by mf-code** [comment]
[5797] - fVcFCWDD2SHC

**Comment by mf-code** [comment]
[5797] - NiHDgoZ5RCSG

**Comment by mf-code** [comment]
[5797] - TIRpxDOYlU5O

**Comment by mf-code** [comment]
[5797] - lvWvUMLqOdQO

**Comment by mf-code** [comment]
[5797] - vCbFnwV0cvkg

**Comment by mf-code** [comment]
[5797] - GRt1BZcp0AGt

**Comment by mf-code** [comment]
[5797] - WgecQFmFjVd5

**Comment by mf-code** [comment]
[5797] - VFIa4xPtXJsS

**Comment by mf-code** [comment]
[5797] - eok2OYevmSRV

**Comment by mf-code** [comment]
[5797] - jm6lakAkcqEE

**Comment by mf-code** [comment]
[5797] - 9MvfOTp05zkh

**Comment by mf-code** [comment]
[5797] - openjdk8 - Allure report on Netlify: https://5f0ed6e352ffe45f2b704c19--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5797] - openjdk10 - Allure report on Netlify: https://5f0ed7d0a777d6206d2d480c--musing-swartz-6fc060.netlify.app

**Pull request merged**